core/src/display_object/edit_text.rs

```diff
@@ -743,7 +743,7 @@ impl<'gc> EditText<'gc> {
         if autosize != AutoSizeMode::None {
             if !is_word_wrap {
                 // The edit text's bounds needs to have the padding baked in.
-                let width = intrinsic_bounds.width() + padding;
+                let width = intrinsic_bounds.extent_x() + padding;
```

Comment:
Based on nosamu's tests, this seems wrong. I have no experience in this part of Ruffle, but this seems like it's just adding padding to fix right alignment, and it breaks center alignment.